README.md

```diff
@@ -481,6 +480,12 @@ As of the time of writing, there are three outcomes to a request made, with resp
   - The request is logged with the prefix "Request made without an API key" including information about the client.
   - The client application should be updated to use a valid API key in future.
 
+#### Documentation
+
+Example POST requests for all the Sequencscape API v2 resources are available to import into Postman REST Client.
+
+The file `Sequencescape API v2.postman_collection.json` is stored in the Pipeline Solutions Shared Network Drive (Finder > Go > Connect to Server > `smb://files-smb/pipeline_solutions`)
```

Comment:
This is a great resource, thanks very much! I tested importing it to Postman and using one of the endpoints.